lnp2p/listen.go

```diff
@@ -89,9 +92,8 @@ func acceptConnections(listener *lndc.Listener, port int, pm *PeerManager) {
 				logging.Errorf("problem saving peer info to DB: %s\n", err.Error())
 			}
 		} else {
-			npeer.nickname = pi.Nickname
-			// TEMP
-			npeer.idx = &pi.PeerIdx
+			// we do have this peer in our db already, don't connect
+			return fmt.Errorf("We already have this peer in our db, returning")
```

Comment:
This would mean that you can only connect one time, and then it would never allow more connections.

Reply:
Corrected, thanks!